packages/core/components/FileThumbnail/FileThumbnail.module.css

```diff
@@ -8,3 +8,11 @@
 .no-thumbnail {
     fill: var(--border-color);
 }
+
+.thumbnailPlaceholder {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--border-color);
+    border: 5px var(--primary-background-color);
```

Comment:
Adding `margin: auto;` here should take care of the centering issue

Reply:
resolved in [dcce3a5](https://github.com/AllenInstitute/biofile-finder/pull/268/commits/dcce3a53988fff6be5dbfdfd0dd0d19fcbda3357)